javamelody-core/src/main/java/net/bull/javamelody/internal/web/html/HtmlReport.java

```diff
@@ -212,62 +210,24 @@ private void writeHtmlHeader(boolean includeSlider, boolean includeCssInline)
 		if (includeSlider) {
 			writeln("<script type='text/javascript' src='?resource=slider.js'></script>");
 		}
+		writeln("<script type='text/javascript' src='?resource=monitoring.js'></script>");

```

Comment:
when includeCssInline above, includes of prototype.js and monitoring.js are missing